tests/conftest.py

```diff
@@ -68,6 +68,7 @@ def app(request):
     Babel(app)
     Mail(app)
     InvenioDB(app)
+    app.debug = True
```

Comment:
As `create_user_for_test` asserts that `debug` is True, yeah, it's needed.
Although it could be moved out of the general app fixture and manually set in the tests that use said function instead.

`TESTING` won't work, as it comes with the side effect that the `@login_required` decorator is ignored.